static/src/javascripts/lib/report-error.spec.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import reportError from './report-error';
+
+describe('report-error', () => {
+    const error = new Error('Something broke.');
+    const meta = {
+        test: true,
+    };
+
+    test('Does not throw an error', () => {
+        expect(() => {
+            reportError(error, meta, false);
+        }).not.toThrowError(error);
+    });
+
+    test('Throws an error', () => {
+        expect(() => {
+            reportError(error, meta);
+        }).toThrowError(error);
+    });
```

Comment:
@SiAdcock @gtrufitt Should we test here aswell, whether `raven.captureException(err, { tags });` is called? If so, how would you do this?